src/sass/components/_header.scss

```diff
@@ -126,6 +126,108 @@ header {
       height: 84px;
```

Comment:
duplicated file!  _header.scss was moved to section directory.
git pull is required

Reply:
resolved